cra-template/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@luigi-project/cra-template-luigi-microfrontend",
+  "version": "0.3.1",
```

Comment:
decide how to manage version number:
- commit to master with new version and release manually
- commit to branch and use that branch to release then merge it afterwards

Reply:
add license file ?